model/h2oautoml/dffml_model_h2oautoml/h2oautoml.py

```diff
@@ -0,0 +1,337 @@
+    Command line usage
+
+    First download the training and test files, change the headers to DFFML
+    format.
+
+    .. code-block::
+        :test:
+
+        $ wget http://download.tensorflow.org/data/iris_training.csv
+        $ wget http://download.tensorflow.org/data/iris_test.csv
+        $ sed -i 's/.*setosa,versicolor,virginica/SepalLength,SepalWidth,PetalLength,PetalWidth,classification/g' iris_training.csv iris_test.csv
+
+
+    Run the train command
+
+    .. code-block:: console
+        :test:
+
+        $ dffml train \
+            -sources train=csv \
+            -source-filename iris_training.csv \
+            -model h2oautoml \
+            -model-features \
+              SepalLength:float:1 \
+              SepalWidth:float:1 \
+              PetalLength:float:1 \
+              PetalWidth:float:1 \
+            -model-predict classification \
+            -model-directory model \
+            -model-max_models 5
+
+    Assess the accuracy
+
+    .. code-block:: console
+        :test:
+
+        $ dffml accuracy \
+            -sources train=csv \
+            -source-filename iris_test.csv \
+            -model h2oautoml \
+            -model-features \
+              SepalLength:float:1 \
+              SepalWidth:float:1 \
+              PetalLength:float:1 \
+              PetalWidth:float:1 \
+            -model-predict classification \
+            -model-directory model
+            
+    Output
+
+    .. code-block::
+
+        accuracy: 0.8685475825
+
+
+    Make predictions
+
+    .. code-block:: console
+        :test:
+
+        $ dffml predict all \
+            -sources train=csv \
+            -source-filename iris_test.csv \
+            -model h2oautoml \
+            -model-features \
+              SepalLength:float:1 \
+              SepalWidth:float:1 \
+              PetalLength:float:1 \
+              PetalWidth:float:1 \
+            -model-predict classification \
+            -model-directory model
+
+    Python usage
+
+    **run.py**
+
+    .. literalinclude:: ../model/h2oautoml/examples/example_h2oautoml.py
+        :test:
+        :filepath: run.py
+
+    Output
+
+    .. code-block::
+        :test:
+
+        $ python run.py
+        Test RMSE: 20.0897
+        Training RMSE: 0.7263
+    """
+
+    # The configuration class needs to be set as the CONFIG property
+    CONFIG = H2oAutoMLModelConfig
+
+    def __init__(self, config) -> None:
+        super().__init__(config)
+
+        # Initialize and connect to h2o
+        h2o.init()
+
+        # The saved model
+        self.saved = None
+        self.saved_filepath = pathlib.Path(self.config.directory)
+
+        # Load saved model if it exists
+        try:
+            path = (
+                str(self.saved_filepath)
+                + "/"
+                + str(os.listdir(str(self.saved_filepath))[1])
+            )
+            print("\n\n", path)
+            self.saved = h2o.load_model(path)
+        except:
+            pass
```

Comment:
We should probably log on error here (self.logger.error or something)

Reply:
Can we check if the path exists first or does that not help us here?

Reply:
@nitesh585 I wanted to get this merged so it doesn't get stale, can you open another PR to fix these small things please?

Reply:
That was a mistake, I should not have merged this, I backed it out the tests weren't setup to run in CI and then failed: https://github.com/pdxjohnny/dffml/tree/h2oautoml

Reply:
Please open another PR

Reply:
@pdxjohnny  sure! I will open another PR.